controllers/usersController.js

```diff
@@ -1,21 +1,36 @@
-const express = require('express');
-const { getAllUsers, getUserById } = require('../queries/usersQueries');
+const express = require("express");
+const { getAllUsers, getUserById } = require("../queries/usersQueries");
 
 const usersController = express.Router();
 
-usersController.get('/', (req, res) => {
-  users = getAllUsers() || [];
-  res.status(200).json({ data: users });
+usersController.get("/", async (req, res) => {
+  try {
+    users = (await getAllUsers()) || [];
+    if (users.length > 0) {
+      res.status(200).json({ data: users });
+      console.log("=== GET users: ", users, "===");
```

Comment:
Be sure to remove `console.log` statements before you commit the code or open a PR